SQL/New_patches/2020-01-20_adding_electrophysiology_browser_to_modules_table.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO modules (Name, Active) VALUES ('electrophysiology_browser', 'Y');
+
```

Comment:
Also not a big deal because the PR was sent before the script was merged, but we should probably just direct people to use the tools/manage_modules script in the release notes.